launcher/src/main/java/org/wso2/carbon/launcher/extensions/model/BundleInfo.java

```diff
@@ -0,0 +1,99 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wso2.carbon.launcher.extensions.model;
+
+/**
+ * A Java class which models a holder for information of an OSGi bundle.
+ *
+ * @since 5.0.0
```

Comment:
I left it as it is because I introduced this class for the first time in 5.0.0. The other classes are slightly changed, but in this only some new methods have been added since then.